readme.md

```diff
@@ -268,31 +304,30 @@ app.get('/users/:userName', (req, res) => {
+app.delete('/users/:userName', (req, res) => {
     const indexOfUserToBeDeleted = users.findIndex(u => u.firstName === req.params.userName);
     users.splice(indexOfUserToBeDeleted, 1);
     res.json(users);
 });
 
-app.post('/users/addUser/:userId&:firstName&:lastName&:title&:email&:taskId', (req, res) => {
-    const newUser = {
-        userId: req.params.userId,
-        firstName: req.params.firstName,
-        lastName: req.params.lastName,
-        title: req.params.title,
-        email: req.params.email,
-        taskId: req.params.taskId,
-    };
+app.post('/users', (req, res) => {
+    const user = req.body();
     users.push(newUser);
     res.json(users);
 })
 
-const port = 4000
+const port = 4000;
 app.listen(port, () => {
     console.log(`Server is listening on port ${port}`);
 });
 ```
-There you go! Your very first, very simple also, Node.js server that uses Express with REST endpoints! It is very easy, there's a lot more you can do with REST, but I guess my target here was to give you a brief idea of what REST is. What we did is similar to what I was exposed to on my adventure to become a software developer, which is still ongoing;) but it was also a huge eye-opener for me as for understanding client-server architecture! However, as I'm sure you can imagine now, you need to expose a different endpoint for each data you want to send, our project has only 2 entities, users and tasks, and it's very plain and simple - imagine how complex REST could be if you add another 10, 20 or more entities which can somehow interact with each other! Imagine that everything we did so far would need to be accessing the database through, for example, SQL statements, that would need to be placed in a different layer of our server to hide business logic! Imagine writing all those REST endpoints for your "dream" application! Not that easy anymore, isn't it?!
 
+## Summary
+
+There you go! Your very first Node.js server that uses Express with REST endpoints. It is very easy, there's a lot more you can do with REST, but I guess my target here was to give you a brief idea of what REST is. What we just did is similar to what I was exposed to on my adventure to become a software developer, which is still ongoing 😉
+
+It was also a huge eye-opener for me as for understanding client-server architecture! However,as I'm sure you can imagine now, you need to expose a different endpoint for each data you want to send, our project has only 2 entities, users and tasks, and it's very plain and simple - imagine how complex REST could be if you add another 10, 20 or more entities which can somehow interact with each other!
+
+Imagine that everything we did so far would need to be accessing the database through, for example, SQL statements, that would need to be placed in a different layer of our server to hide business logic! Imagine writing all those REST endpoints for your "dream" application! Not that easy anymore, is it?!
 
```

Comment:
I'd recommend adding some links to useful resources that you would recommend to learn more. What did you use to learn REST?